src/classes/HelloPortland.cls

```diff
@@ -1,5 +1,11 @@
 public with sharing class HelloPortland {
 	public static String sayHello(String name) {
-		return String.format('Hello {0} !', new List<String> {name});
+		return String.format('Hello {0}!', new List<String> {name});
+	}
+
+	public static void badThing(Integer i) {
+		for (Integer j=0; j<i; j++) {
+			insert new Account(Name = 'New Account ' + i);
```

Comment:
This DML statement is inside a loop and is highly likely to hit governor limits. Please use collections to move the DML statement outside of the loop.